bin/build_image.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# build the portal Docker image and tag accordingly, then push to GCR
+# cleans up untagged images after push when overwriting tags
```

Comment:
Your PR description here seems like it could help maintainers down the road.

```suggestion
# cleans up untagged images after push when overwriting tags; run in CI/CD
# More context: https://github.com/broadinstitute/single_cell_portal_core/pull/1552
```